libraries/AP_HAL_ChibiOS/hwdef/common/ffconf.h

```diff
@@ -247,7 +261,7 @@
 
 
 #define FF_FS_REENTRANT   0
-#define FF_FS_TIMEOUT     chTimeMS2I(1000)
+#define FF_FS_TIMEOUT     MS2ST(1000)
```

Comment:
so ChibiOS rolled back to previous method 😄

Reply:
it is an incorrect change, but harmless as only used when FF_FS_REENTRANT is 1
I'll fix